modules/ROOT/pages/component/remove.adoc

```diff
@@ -0,0 +1,109 @@
+= Remove a component
+:description: This section describes how to remove components from Neo4j Ops Manager.
+
+[IMPORTANT]
+====
+Metrics related to a component that has been removed (either directly or indirectly) will be removed as part of the scheduled daily clean up process for metrics.
+They will not be visible once the component has been removed but may remain in the persistence storage until the clean up process has run. 
+====
+
+
+[[agent]]
+== Agent
+
+=== Conditions
+It is only possible to remove agents that have either never contacted the server or more time than the 'agent cleanup safe-period' has passed since the last contact. 
```

Comment:
```suggestion
Only agents that have never contacted the server or where the `agent cleanup safe-period` has been exceeded since last contact, can be removed.
```
Provided that it is a code block you want?